net-vpn/i2pd/i2pd-2.33.0.ebuild

```diff
@@ -3,49 +3,40 @@
 
 EAPI=7
 
-inherit systemd cmake toolchain-funcs
+inherit cmake toolchain-funcs systemd
 
 DESCRIPTION="A C++ daemon for accessing the I2P anonymous network"
 HOMEPAGE="https://github.com/PurpleI2P/i2pd"
 SRC_URI="https://github.com/PurpleI2P/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~x86"
-IUSE="cpu_flags_x86_aes cpu_flags_x86_avx i2p-hardening libressl static +upnp websocket"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="cpu_flags_x86_aes cpu_flags_x86_avx i2p-hardening libressl static +upnp"
 
-# if using libressl, require >=boost-1.65, see #597798
 RDEPEND="
 	acct-user/i2pd
 	acct-group/i2pd
 	!static? (
 		dev-libs/boost:=[threads]
 		!libressl? ( dev-libs/openssl:0=[-bindist] )
-		libressl? (
-			dev-libs/libressl:0=
-			>=dev-libs/boost-1.65:=
-		)
+		libressl? ( dev-libs/libressl:0= )
 		upnp? ( net-libs/miniupnpc )
 	)"
 DEPEND="${RDEPEND}
 	static? (
```

Comment:
Does anyone actually want static for this?

Reply:
That's hard to tell. My feeling is that most people who use i2p don't want to be using it publicly. Maybe it can be useful for embedded devices?

Reply:
This was kind of my thinking too. So, good enough for now. :)